### PR TITLE
Create group of active phd students FIST-369 #resolves

### PR DESCRIPTION
**Pull request by fernandojps**

### CODE REVIEW

File: fenixedu-ist-integration/src/main/resources/resources/FenixeduIstIntegrationResources_en.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
 authorize.personal.data.access.title = Manage Data Access
 authorize.personal.data.access.title.bpi = Student Union Card / BPI
 authorize.personal.data.access.title.cgd = ULisboa Card / SAS / CGD
-authorize.personal.data.access.title.santander = T�cnico Identification Card / Santander
-authorize.personal.data.access.description.bpi = Grant Banco Portugu�s de Investimento access to my personal information (name, address, contact information, identification number, photograph and registration information) to possibly open an account in that bank and to produce my Student Union Identification Card?
-authorize.personal.data.access.description.cgd = Grant Caixa Geral de Dep�sitos access to my personal information (name, address, contact information, identification number, photograph and registration information) to possibly open an account in that bank and to produce my University Identification Card, <b>required for access to IST's canteen</b>, as well as a few other services provided by the university?
+authorize.personal.data.access.title.santander = T\u00E9cnico Identification Card / Santander
```
Comment (pedrosan7os) [inline fenixedu-ist-integration/src/main/resources/resources/FenixeduIstIntegrationResources_en.properties]: There are some encoding issues in this PR

**Comment by fernandojps** [comment]
I've fixed the encoding issues.

Would a good solution be to update the activeStudents group with an optional argument for PHD students? I also noticed that the ActiveStudentsGroup class is part of the Academic Module, while this issue is FIST, could I add a PR with the mentioned change using the same issue id on the fenixedu-academic repository?

**Comment by sergiofbsilva** [comment]
The idea of this issue is not to change ACDM but FIST instead because we should not perpetuate phd related code.

Since refactoring phd is a epic journey, this just mitigates the users' problems regarding phd active students. 

@pedrosan7os if you think academic should be changed, this issue should be reviewed and moved to academic. your call.

**Comment by pedrosan7os** [comment]
Why does not `student(cycle=THIRD_CYCLE)` have the semantic you need?

**Comment by sergiofbsilva** [comment]
because that will only happen for all phd students when all of them have a registration which is not the case when there is no curricular plan associated to the phd process. This will be fixed after refactoring phd related code. 

Meanwhile a specific group that checks the phd process is required to fullfill user needs.

**Comment by pedrosan7os** [comment]
There is no group that captures PHDs that do not have a registration, but that is an absolute mess that has to end. I'd rather avoid coding any more for that condition, and move towards have a registration for every phd.

Coding in another module does not help, it maintains the academic project clean but messes up another one, and one that has nothing to do with this mess.

We handle the mess in data, change the recipients group to activeStudents - student(cycle=FIRST_CYCLE) - student(cycle=SECOND_CYCLE) - student(degree_type=DEA). It's messy but works and changes no code. Also change the label to reflect that we are talking to phd students and not to DEA students.

**Pull request closed without merging**